src/main/java/com/elasticbox/jenkins/util/ClientCache.java

```diff
@@ -71,6 +73,17 @@ public static final Client findOrCreateClient(String cloudName) throws IOExcepti
         return client;
     }
 
+    public static HttpProxy getHttpProxy() {
```

Comment:
I would rename it to something more meaningful, such as:
```suggestion
    public static HttpProxy getJenkinsHttpProxyCfg() {
```

Reply:
Fixed up